### DISCUSSION TITLE
Build Errors in Vercel | Caching issue?

### DISCUSSION THREAD

**Discussion by KoesterJannik** [vercel/community#720] [Help] [answered]
Hello,

today i faced some weired issues. I added a new prisma model, tested everything and deployed it using vercel --prod

And I always got this error 
```Type error: Property 'subscription' does not exist on 
type 'PrismaClient<PrismaClientOptions, never, RejectOnNotFound | RejectPerOperation | undefined>'.
```

I solved it by creating a new vercel project and then i deployed my code there. Now everything works.

So my question is: Is Vercel caching something  or is there a way to clean up the old stuff if a new `vercel --prod` gets called? 
I dont want to create a new project everytime i make a database change.

Maybe you face this aswell. 

Thanks!

**Comment by swarnava** [comment, marked as answer]
Hi there,

You can also use `vercel --force` flag to clear cache if this is something causing trouble. it can be used together such as `vercel --prod --force`.

Ideally, deploying using `--prod` should build without cache.
